client/src/Router.tsx

```diff
@@ -20,7 +20,7 @@ import { AppSettingsPorts } from './pages/app/settings/ports';
 import { AppSettingsDomains } from './pages/app/settings/domains';
 import { AppSettingsAdvanced } from './pages/app/settings/advanced';
 
-const PrivateRoute = ({ children, ...rest }: RouteProps) => {
+const PrivateRoute = ({ children, ...rest }: any) => {
```

Comment:
Can we keep using `RouteProps ` here?

Reply:
I wasn't able to, `RouteProps` gave me following error:

![RouteProps-render-issue](https://user-images.githubusercontent.com/24852597/188509724-30821eec-1557-4ea9-8477-649ef5b6982b.png)

Switching to `any` was the best fix I could come up with,   
however I'd gladly take any advice for improvements here!